### ISSUE TITLE
TypeError: CleanWebpackPlugin is not a constructor

### ISSUE THREAD

**Issue by lvfxbr** [johnagan/clean-webpack-plugin#143]
### Issue description or question

In the docs is wrong the way get the plugin variable, it's distructuring the module unnecessarily:
```
const { CleanWebpackPlugin } = require('clean-webpack-plugin');
```

Doing that way, triggers this error:
```
TypeError: CleanWebpackPlugin is not a constructor
```
The right is:
```
const CleanWebpackPlugin = require('clean-webpack-plugin');
```
### Webpack Config

```js
const path = require('path');

const CleanWebpackPlugin = require('clean-webpack-plugin');

module.exports = {
    mode: 'development',
    entry: {
        newtab: './src/newtab.js'
    },
    output: {
        filename: '[name].js',
        path: path.resolve(__dirname, 'dist')
    },
    plugins: [
        new CleanWebpackPlugin(),
    ],
};
```

### Environment

Run: `npx envinfo --system --binaries --npmPackages clean-webpack-plugin,webpack`

```bash
  System:
    OS: Windows 10
    CPU: (4) x64 Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
    Memory: 3.67 GB / 7.88 GB
  Binaries:
    Node: 10.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.13.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.4.1 - C:\Program Files\nodejs\npm.CMD
```
Please, test the things before put in the docs and test the installation of module before publish it to Github because it can confuse the users, thanks!

**Comment by gustawdaniel** [comment]
I have the same error.

**Comment by gustawdaniel** [comment]
Other solution is change default displayed branch from master to `lts` or `last stable release` or something like this because of almost all developers uses `README.md` on main page as source of knowledge about last released version.

**Comment by VisonM** [comment]
@gustawdaniel it's right
`const CleanWebpackPlugin = require('clean-webpack-plugin');`

**Comment by chrisblossom** [comment]
It is normal for a github project's readme to be out of date with what is published on npm. Either use [github tags](https://github.com/johnagan/clean-webpack-plugin/tree/v2.0.2) or view the readme from the [npm page](https://www.npmjs.com/package/clean-webpack-plugin).

**Comment by johnagan** [comment]
released fixed for in `v3.0.0`
